packages/component-utils/src/mineralTheme.js

```diff
@@ -111,5 +111,7 @@ export default {
   color_text_success: color.green_90,
   color_text_onsuccess: 'white',
   color_text_warning: color.ochre_90,
-  color_text_onwarning: 'white'
+  color_text_onwarning: 'white',
+
+  baseFontSize: '16px'
```

Comment:
Good call. Additionally, I moved it out of mineralTheme and into styleVariables, because all of things affected by `fontSize_base` are in the latter.